components/cors-mgt/org.wso2.carbon.identity.cors.mgt.core/src/test/resources/dbscripts/config/h2.sql

```diff
@@ -1,60 +1,112 @@
-  ADD CONSTRAINT RESOURCE_ID_ATTRIBUTE_FOREIGN_CONSTRAINT FOREIGN KEY (RESOURCE_ID) REFERENCES IDN_CONFIG_RESOURCE (ID) ON DELETE CASCADE ON UPDATE CASCADE;
+    ADD CONSTRAINT RESOURCE_ID_ATTRIBUTE_FOREIGN_CONSTRAINT FOREIGN KEY (RESOURCE_ID) REFERENCES IDN_CONFIG_RESOURCE (ID) ON DELETE CASCADE ON UPDATE CASCADE;
 
--- -----------------------------------------------------
--- Table IDN_CONFIG_FILE
--- -----------------------------------------------------
 CREATE TABLE IF NOT EXISTS IDN_CONFIG_FILE (
-  ID        VARCHAR(255) NOT NULL,
-  VALUE     BLOB         NULL,
-  NAME        VARCHAR (255) NULL,
-  RESOURCE_ID VARCHAR(255) NOT NULL,
-  PRIMARY KEY (ID)
+    ID        VARCHAR(255)    NOT NULL,
+    VALUE     BLOB            NULL,
+    NAME        VARCHAR (255) NULL,
+    RESOURCE_ID VARCHAR(255)  NOT NULL,
+
+    PRIMARY KEY (ID)
 );
 ALTER TABLE IDN_CONFIG_FILE
-  ADD CONSTRAINT RESOURCE_ID_FILE_FOREIGN_CONSTRAINT FOREIGN KEY (RESOURCE_ID) REFERENCES IDN_CONFIG_RESOURCE (ID) ON DELETE CASCADE ON UPDATE CASCADE;
+    ADD CONSTRAINT RESOURCE_ID_FILE_FOREIGN_CONSTRAINT FOREIGN KEY (RESOURCE_ID) REFERENCES IDN_CONFIG_RESOURCE (ID) ON DELETE CASCADE ON UPDATE CASCADE;
+
+CREATE TABLE IF NOT EXISTS SP_APP (
+    ID INTEGER NOT NULL AUTO_INCREMENT,
+    TENANT_ID INTEGER NOT NULL,
+    APP_NAME VARCHAR (255) NOT NULL ,
+    USER_STORE VARCHAR (255) NOT NULL,
+    USERNAME VARCHAR (255) NOT NULL ,
+    DESCRIPTION VARCHAR (1024),
+    ROLE_CLAIM VARCHAR (512),
+    AUTH_TYPE VARCHAR (255) NOT NULL,
+    PROVISIONING_USERSTORE_DOMAIN VARCHAR (512),
+    IS_LOCAL_CLAIM_DIALECT CHAR(1) DEFAULT '1',
+    IS_SEND_LOCAL_SUBJECT_ID CHAR(1) DEFAULT '0',
+    IS_SEND_AUTH_LIST_OF_IDPS CHAR(1) DEFAULT '0',
+    IS_USE_TENANT_DOMAIN_SUBJECT CHAR(1) DEFAULT '1',
+    IS_USE_USER_DOMAIN_SUBJECT CHAR(1) DEFAULT '1',
+    ENABLE_AUTHORIZATION CHAR(1) DEFAULT '0',
+    SUBJECT_CLAIM_URI VARCHAR (512),
+    IS_SAAS_APP CHAR(1) DEFAULT '0',
+    IS_DUMB_MODE CHAR(1) DEFAULT '0',
+    UUID CHAR(36),
+    IMAGE_URL VARCHAR(1024),
+    ACCESS_URL VARCHAR(1024),
+    IS_DISCOVERABLE CHAR(1) DEFAULT '0',
+
+    PRIMARY KEY (ID));
+
+CREATE TABLE IF NOT EXISTS IDN_CORS_ORIGIN (
+    ID                INT           NOT NULL AUTO_INCREMENT,
+    TENANT_ID         INT           NOT NULL,
+    ORIGIN            VARCHAR(2048) NOT NULL,
+    IS_TENANT_LEVEL   CHAR(1)       NOT NULL DEFAULT '0',
+
+    PRIMARY KEY (ID),
+    UNIQUE (TENANT_ID, ORIGIN)
+);
+
+CREATE TABLE IF NOT EXISTS IDN_CORS_ASSOCIATION (
+    IDN_CORS_ORIGIN_ID  INT NOT NULL,
+    SP_APP_ID           INT NOT NULL,
+
+    PRIMARY KEY (IDN_CORS_ORIGIN_ID, SP_APP_ID)
+);
+
+ALTER TABLE IDN_CORS_ASSOCIATION
+    ADD CONSTRAINT ID_IDN_CORS_ORIGIN_ID_FOREIGN_CONSTRAINT
+        FOREIGN KEY (IDN_CORS_ORIGIN_ID)
+        REFERENCES IDN_CORS_ORIGIN (ID)
+        ON DELETE CASCADE
+        ON UPDATE RESTRICT;
+
+--ALTER TABLE IDN_CORS_ASSOCIATION
+--    ADD CONSTRAINT SP_APP_ID_ID_FOREIGN_CONSTRAINT
```

Comment:
Can we remove the commented part?

Reply:
The original SQL related to the CORS has this foreign key. But we can't do it here because we don't have the SP table in the CORS tests. I thought it is better to keep it as it shows that there is something different with the real schema. WDYT?

Reply:
Ok. But putting a comment mentioning that will be enough IMO. No need to add the full schema as a comment..